plutus-playground-server/app/PSGenerator.hs

```diff
@@ -263,23 +262,23 @@ mySettings =
     (defaultSettings & set apiModuleName "Playground.Server")
         {_generateSubscriberAPI = False}
 
-multilineString :: BS.ByteString -> BS.ByteString -> BS.ByteString
+multilineString :: T.Text -> T.Text -> T.Text
```

Comment:
nice

Reply:
Yeah I did a bunch of drive-by text stuff, I think this is also maybe a little more correct because it's using encode/decodeUtf8.